assets/js/api/elasticSearchUtils.js

```diff
@@ -18,11 +18,26 @@ export const buildFilter = (filters) => {
 
 export const buildSort = (sortBy) => {
   if (sortBy === ALPHABETICAL) {
-    return [
-      {
-        'publiccode.description.it.localizedName.keyword': { order: 'asc', unmapped_type: 'keyword' },
+    return {
+      _script: {
+        type: 'string',
+        order: 'asc',
+        script: {
+          lang: 'painless',
+          source: `
+             if (
+                 doc['publiccode.description.it.localisedName.keyword'].size() != 0
```

Comment:
I'm personally against this kind of manipulation but it can be useful to have current language as a parameter and fallback (as already done) whether it is not present in publiccode. wdyt?